app/src/main/java/com/kickstarter/libs/SegmentTrackingClient.kt

```diff
@@ -0,0 +1,38 @@
+package com.kickstarter.libs
+
+import android.content.Context
+import com.segment.analytics.Analytics
+import com.segment.analytics.Properties
+
+class SegmentTrackingClient(
```

Comment:
nit: Could we alphabetize this list of dependencies?

Reply:
Done